src/models/UsersModel.js

```diff
@@ -0,0 +1,51 @@
+class UsersModel {
+    constructor({
+      name,
+      email,
+      profilePicture = 'http://localhost:8000/images/userProfileImage.jpg', // Default image URL
+      loginType = null,
+      adminCode = null,
```

Comment:
I don't think we need to store an admin code on the user. The code is a universal constant that should be used in the signup route to determine if the created user object should have `isAdmin` set to true or false.